src/App.js

```diff
@@ -1,29 +1,23 @@
 import React from 'react';
+import { PostList } from './api/components/PostList/PostList';
 
 import './App.scss';
 
 import posts from './api/posts';
 import comments from './api/comments';
 import users from './api/users';
 
+const preparedPosts = posts.map(post => ({
+  ...post,
+  user: users.find(user => user.id === post.userId),
+  comment: comments.filter(comment => comment.postId === post.id),
```

Comment:
Better to use more appropriate names here.

```suggestion
  commentList: comments.filter(comment => comment.postId === post.id),
```